packages/generators/README.md

```diff
@@ -0,0 +1,9 @@
+# Generator
+
+Generate a sub-package under wp-calypso
```

Comment:
This is a great place to mention the Calypso monorepo and maybe link to its docs: 

https://github.com/Automattic/wp-calypso/blob/trunk/docs/monorepo.md#L1

Reply:
Added reference you mentioned to the README.md